mods/cnc/rules/structures.yaml

```diff
@@ -200,7 +206,7 @@ PROC:
 		PipCount: 10
 		Capacity: 700
 	Selectable:
-		Bounds: 72,56,0,12
+		Bounds: 72,56
```

Comment:
Why remove the offset here?

Reply:
See description of https://github.com/OpenRA/OpenRA/pull/15198/commits/77603d23d52b8de77e27f7c04e294d621ed5bfb4 for my reasoning. If you disagree I can drop this change, but to me the new value feels better.